.github/workflows/post-merge-container-update.yml

```diff
@@ -0,0 +1,92 @@
+name: Post-merge container update
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    branches: [ "main" ]
+    paths: ['tripy/Dockerfile', 'tripy/pyproject.toml']
+  # TODO: For testing purposes - remove before merging
```

Comment:
Why remove before merging? We want to build container only when these file changes right?

Reply:
I only want to push the container to the registry after changes have merged. We don't want containers from in-flight PRs (which may never be merged) to be published. We could introduce a separate workflow to push the temporary containers that might be needed for testing.